addons/fire/XEH_postInit.sqf

```diff
@@ -13,6 +13,19 @@
     TRACE_1("settingsInit", GVAR(enabled));
     if (!GVAR(enabled)) exitWith {};
 
+    ["AllVehicles", "Killed", {
+        params ["_vehicle", "_killer"];
+
+        if !(local _vehicle) exitWith {};
+
+        if (getText (configOf _vehicle >> "destrType") == "") exitWith {};
```

Comment:
I think this will also add a fire source for destroyed buildings. Probably best to explicitly define the vehicles which this happens rather than assume that you will get all of the edge cases